src/main/java/group1/mips_simulator/components/README.md

```diff
@@ -0,0 +1,2 @@
+## Emulation Component of Simulator
+
```

Comment:
Whats the plan with this README.md file? Will it be filled out later? Or should this be removed?

Reply:
Oh I see, this is a hold over from the other branch. See my comment on that other MR then :)